internal/compiler/passes.rs

```diff
@@ -69,6 +69,15 @@ pub async fn run_passes(
             .unwrap_or_else(|| panic!("can't load style metrics"))
     };
 
+    let palette = {
+        // Ignore import errors
+        let mut build_diags_to_ignore = crate::diagnostics::BuildDiagnostics::default();
+        type_loader
+            .import_component("std-widgets.slint", "Palette", &mut build_diags_to_ignore)
```

Comment:
We've been using the `StyleMetrics` global before in the compiler to "communicate" between the style and the built-ins.
I feel it would be better to keep everything in the StyleMetrics for consistency.
But I know this is currently not in the StyleMetrics. But then we should also use the Palette for all the colors instead of the StyleMetrics.
So I wonder if it adds technical depth to use Palette, or if using StyleMetrics would increase technical depth.